### PR TITLE
遷移先の指定をtrueからfalseに変更しました

### PR DESCRIPTION
**Pull request by SekiMoemi**

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java [hunk 1]
```diff
@@ -29,11 +29,11 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 				.permitAll().anyRequest().authenticated());
 		// ログインに関する設定
 		http.formLogin(login -> login.loginPage("/login/toLogin").loginProcessingUrl("/login")
-				.failureUrl("/login/toLogin?error=true").defaultSuccessUrl("/", true)
+				.failureUrl("/login/toLogin?error=true").defaultSuccessUrl("/", false)
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:32]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 102).

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:32]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java [hunk 2]
```diff
@@ -29,11 +29,11 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
...
 				.usernameParameter("email").passwordParameter("password"));
 		// ログアウトに関する設定
 		http.logout(logout -> logout.logoutRequestMatcher(new AntPathRequestMatcher("/logout**"))
-				.logoutSuccessUrl("/showItemList/showItemList").deleteCookies("JSESSIONID")
+				.logoutSuccessUrl("/").deleteCookies("JSESSIONID")
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:36]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java [hunk 3]
```diff
@@ -43,7 +43,7 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 	 */
 	@Bean
 	public WebSecurityCustomizer webSecurityCustomizer() {
-		return web -> web.ignoring().requestMatchers("/img/**", "/js/**", "/css/**");
+		return web -> web.ignoring().requestMatchers("/js/**", "/css/**", "/img/**");
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:46]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:46]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

**Review by yamada-kensyu — APPROVED** [review]
承認します！ありがとうございます

**Pull request merged**